pkg/daemon/cleanup/cleanup.go

```diff
@@ -33,30 +35,41 @@ import (
 	"github.com/openshift/windows-machine-config-operator/pkg/daemon/winsvc"
 	"github.com/openshift/windows-machine-config-operator/pkg/nodeconfig"
 	"github.com/openshift/windows-machine-config-operator/pkg/servicescm"
+	"github.com/openshift/windows-machine-config-operator/pkg/windows"
 )
 
 // Execute removes an instance from the cluster. If the parameter flag is true, the node object is not deleted.
 // If we are able to get the services ConfigMap tied to the desired version, all services defined in it are cleaned up.
-// TODO: Best effort cleanup if node object, desired version annotation, or services ConfigMap are not attainable
+// Otherwise, perform cleanup based on a combination of the OpenShift managed tag and the latest services ConfigMap.
 func Execute(c client.Client, addrs []net.Addr, svcMgr manager.Manager, preserveNode bool) error {
 	ctx := context.TODO()
 	node, err := controller.GetAssociatedNode(c, addrs)
 	if err != nil {
-		klog.Exitf("could not node object associated with this instance: %s", err.Error())
+		// If the node object is not attainable use a best effort cleanup stategy
+		klog.Warning(err.Error())
```

Comment:
Since this is a warning, it would be good to add that the ConfigMap based cleanup will be skipped.